src/test/java/net/datafaker/providers/base/LocalityTest.java

```diff
@@ -82,6 +84,19 @@ void localeStringWithoutReplacement() {
         }
     }
 
+    @Test
+    void localeStringWithoutReplacementWithCustomRandomGenerator() {
+        RandomGenerator random = new SplittableRandom();
+        // loop through all supported locales
+        for (int i = 0; i < 2; i++) {
```

Comment:
Why twice?

Reply:
this is just a copy with a different rng